### DISCUSSION TITLE
Tasks?

### DISCUSSION THREAD

**Discussion by sebkolind** [davidmyersdev/octo#127] [Q&A] [answered]
I just started using Octo, but I am confused about how to make something appear under "Tasks"?

**Comment by davidmyersdev** [comment, marked as answer]
Hey @sebkolind! Tasks are documents that contain at least one _unchecked_ markdown checkbox (e.g. `- [ ]`).

### Doc example

```markdown
# Some tasks

- [ ] I need to do the thing
- [x] I have already done the thing
```

**Reply by sebkolind** [reply]
Cool, thanks! There is a lack in documentation - is that on the roadmap?

**Reply by davidmyersdev** [reply]
Very good point, @sebkolind! It is now!
